src/modules/aws-ec2/termination-protection/disableTerminationProtection.ts

```diff
@@ -0,0 +1,4 @@
+import { changeTerminationProtection } from "./changeTerminationProtection";
+export const disableTerminationProtection = (instanceId: string) => {
+  changeTerminationProtection(instanceId, "false");
```

Comment:
If this is async, consider returning it so it can be properly awaited by the caller.